elections/2020/governance-committee-election.md

```diff
@@ -0,0 +1,180 @@
+
+One of two options will be available in the form to prove eligibility:
+
+* GitHub handle
+* List either…
+  * Your contributions to OpenTelemetry that make you eligible to vote, or
+  * Your non-code contributions per the [Members of Standing section of the
+charter](https://github.com/open-telemetry/community/blob/master/governance-charter.md#members-of-standing):
+particularly, your relationship to a well-known project that’s taken a hard
+dependency on OpenTelemetry, and/or your involvement with a well-known
+organization’s adoption of OpenTelemetry as an end-user.
+
+The link to the pre-registration form: [NEED TO UPDATE BEFORE MERGE](http://localhost/)
+
+Note that all pre-registrations will be verified by the Governance Committee
+for eligibility.
+
+All pre-registrations are private. Only the members of a current governance
+committee and election committee will have access to this information. Lists
+will be discarded one month after the election.
+
+# Vote
+
+Everyone with voting rights will receive the link to the voting site.
+[Condorcet Internet Voting Service](https://civs.cs.cornell.edu/) (CIVS) – a
+ranked-choice voting system – will be used for the voting process.
+* The Condorcet-IRV algorithm will be used to calculate the results
+* The “No Opinion” option will be enabled
+
+Late registrations to vote and requests to re-send the voting link will be
+accepted via email to
+[cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io).
+We encourage pre-registration to minimize the effort required to run this
+election.
+
+Per CIVS, voting is completely private: nobody will know any individual's vote.
+
+# Results
+
+Voting will close at 12:01am GST on 2020/10/28. Nominees will be stack ranked.
+If a nominee becomes ineligible (for instance, if more than 2 topmost nominees
+work for the same company), those nominees will be skipped and the nominee with
+the next-highest score will be picked.
+
+Exact scores for each candidate will NOT be made public (unlike the 2019
+election).
+
+
+# Schedule
+
+* September 30: this document announced
+* October 7: Call for nominations and voters pre-registration
+* October 14, 12:01am GST: End of call for nominations
+* October 15: Preliminary list of nominees announced
+* Nominees ratified and Governance Committee ensures nominees’ commitment
+* October 19: List of nominees is finalized on [community](https://github.com/open-telemetry/community) GitHub and advertised via mailing list and Gitter
+* October 25: End of call for voters pre-registration (registration during the voting period is permitted, but highly discouraged)
+* October 26, 12:01am GST: Emails with voting links sent via [CIVS](https://civs.cs.cornell.edu/). Voting period begins
+* October 28, 12:01am GST: Voting ends
+* October 30: Results are announced
+
+# Appendix
+
+We identified active contributors. For these contributors we found at least one
```

Comment:
we need to as Lucasz when he can generate the list for us and declare this date. So we count eligibility based on number of contributions by specific date and exactly one year back.

Reply:
ok cool. I guess we don't formally need that for the first phase of this (that starts tomorrow) which is about getting candidates together.